src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSource.cs

```diff
@@ -621,8 +628,13 @@ private static SuggestedActionSetPriority GetSuggestedActionSetPriority(CodeActi
                     // should be higher in the list when the caret is on a parameter, vs the 
                     // code-fix for "use expression body" which is given the entire span of a 
                     // method.
+
+                    var priority = selection.Length > 0
```

Comment:
so this is saying selection based refactoring is higher priority than caret based refactoring? problem is that a lot of refactoring that is caret based return refactoring even if selection is given. so this will just make same refactoring to have different priority depends on whether it got called with selection or not.

is that the intention?

Reply:
Yes.  i believe that's reasonable behavior.  We can certainly tweak our refactorings/analyzers (as we have in the past) if this ends up causing any unexpected changes.